desktop/hardened-desktop/admin-controls/index.md

```diff
@@ -0,0 +1,55 @@
+---
+description: admin controls for desktop
+keywords: admin controls, rootless, docker desktop, hardened desktop
+title: What is Admin Controls?
+--- 
+>Note
+>
+>Admin Controls is available to Docker Business customers only. 
+
+Admin Controls is a feature that gives administrators the ability to configure certain Docker Desktop settings for their organization. 
+
+With a few lines of JSON, administrators can configure Docker Desktop settings for HTTP proxies, network settings, and the Docker Engine configuration. This saves significant time and cost in securing developer workflows. If administrators also enable [Enhanced Container Isolation](../enhanced-container-isolation/index.md), it ensures that any configurations set with Admin Controls cannot be modified by containers.
```

Comment:
'...settings for features such as proxies, network settings and the Docker Engine configuration (for the full list, see 'What can be set ?' below)

Reply:
can we remove the sentence about 'This saves significant time...workflows' and instead have something at the end of the previous sentence such as ", and the Docker Engine configuration, sparing significant effort in securing developer workflows"

not sure if 'sparing significant effort' is the right wording !